csharp/ql/test/TestUtilities/InlineFlowTest.qll

```diff
@@ -58,42 +55,60 @@ private predicate defaultSink(DataFlow::Node sink) {
 }
 
 module DefaultFlowConfig implements DataFlow::ConfigSig {
```

Comment:
May as well make this private.

Reply:
That definitely possible for C#, but not for some of the other languages, and I'd like to keep this in sync.